package.json

```diff
@@ -15,11 +15,15 @@
   "packageManager": "yarn@4.2.2",
   "resolutions": {
     "ansi-regex": "5.0.1",
+    "cheerio": "1.0.0-rc.12",
     "diff": "3.5.0",
     "glob-parent": "5.1.2",
     "json-schema": "0.4.0",
+    "minimatch": "3.1.2",
     "minimist": "1.2.6",
     "node-fetch": "2.6.7",
+    "semver": "7.7.3",
```

Comment:
Semver was upgraded to 7.7.3 to address ReDoS vulnerabilities while maintaining compatibility across all packages that depend on it.